### PR TITLE
remove this link syntax

### PR DESCRIPTION
**Pull request by brewn**
remove all :doc: links

### CODE REVIEW

File: chef_master/source/aws_opsworks_chef_automate.rst [hunk 1]
```diff
@@ -78,11 +78,11 @@ Enabling the compliance profile storage service in AWS OpsWorks for Chef Automat
 Adding push jobs server based build nodes to AWS OpsWorks for Chef Automate
 ============================================================================
 
-Build nodes enable you to push infrastructure or application changes through a pipeline. Pipelines are part of Chef Automate's :doc:`workflow <workflow>` feature. Build nodes run jobs, called *phases*, which define how your change is built, tested, and deployed to your infrastructure.
+Build nodes enable you to push infrastructure or application changes through a pipeline. Pipelines are part of Chef Automate's `workflow <workflow.html>`__ feature. Build nodes run jobs, called *phases*, which define how your change is built, tested, and deployed to your infrastructure.
```
Comment (davidwrede) [inline chef_master/source/aws_opsworks_chef_automate.rst]: looks like link is missing preceding `/` before the filename.

File: chef_master/source/aws_opsworks_chef_automate.rst [hunk 2]
```diff
@@ -78,11 +78,11 @@ Enabling the compliance profile storage service in AWS OpsWorks for Chef Automat
...
 
-:doc:`Push jobs <push_jobs>` are one way to trigger jobs to run on your build nodes. Configuring push jobs on AWS OpsWorks for Chef Automate requires changes to the Chef Automate instance as well as the node that will run as a builder.
+`Push jobs <push_jobs.html>`__ are one way to trigger jobs to run on your build nodes. Configuring push jobs on AWS OpsWorks for Chef Automate requires changes to the Chef Automate instance as well as the node that will run as a builder.
```
Comment (davidwrede) [inline chef_master/source/aws_opsworks_chef_automate.rst]: same comment as above

File: chef_master/source/aws_opsworks_chef_automate.rst [hunk 3]
```diff
@@ -78,11 +78,11 @@ Enabling the compliance profile storage service in AWS OpsWorks for Chef Automat
...
 
-If you don't already have a system set up to run as your build node, refer to the :doc:`installation guide <install_chef_automate>` to learn about the supported platforms and network requirements. Then, bring up an instance to serve as your build node. The examples that follow use CentOS 7.3 running on Amazon Web Services with ports 22 (SSH) and 443 (HTTPS) open to inbound network traffic.
+If you don't already have a system set up to run as your build node, refer to the `installation guide <install_chef_automate.html>`__ to learn about the supported platforms and network requirements. Then, bring up an instance to serve as your build node. The examples that follow use CentOS 7.3 running on Amazon Web Services with ports 22 (SSH) and 443 (HTTPS) open to inbound network traffic.
```
Comment (davidwrede) [inline chef_master/source/aws_opsworks_chef_automate.rst]: same comment as above

File: chef_master/source/aws_opsworks_chef_automate.rst [hunk 4]
```diff
@@ -242,7 +242,7 @@ This certificate file name appears as ``CHEF_AUTOMATE_SSL_CERT`` in the steps th
 
       build-node-1	available
 
-Because Chef server and Chef Automate exist on the same system, Chef Automate can communicate directly with the Chef server to dispatch push jobs to build nodes. Although not typically required, you can perform the following steps if you would like to use :doc:`knife job <plugin_knife_push_jobs>` to submit push jobs to your build nodes directly.
+Because Chef server and Chef Automate exist on the same system, Chef Automate can communicate directly with the Chef server to dispatch push jobs to build nodes. Although not typically required, you can perform the following steps if you would like to use `knife job <plugin_knife_push_jobs.html>`__ to submit push jobs to your build nodes directly.
```
Comment (davidwrede) [inline chef_master/source/aws_opsworks_chef_automate.rst]: same comment as above

File: chef_master/source/berkshelf.rst [hunk 5]
```diff
@@ -76,7 +76,7 @@ A Berksfile is a Ruby file, in which sources, dependencies, and options may be s
 Source Keyword
 +++++++++++++++
 
-A source defines where Berkshelf should look for cookbooks. Sources are processed in the order that they are defined in, and processing stops as soon as a suitable cookbook is found. Locations include a private or public :doc:`Supermarket <supermarket>`, Chef Server, or local Chef repository.
+A source defines where Berkshelf should look for cookbooks. Sources are processed in the order that they are defined in, and processing stops as soon as a suitable cookbook is found. Locations include a private or public `Supermarket <supermarket.html>`__, Chef Server, or local Chef repository.
```
Comment (davidwrede) [inline chef_master/source/berkshelf.rst]: missing preceding `/` in link

File: chef_master/source/data_collection_habitat.rst [hunk 6]
```diff
@@ -3,4 +3,4 @@ Data Collection with Habitat
 `[edit on GitHub] <https://github.com/chef/chef-web-docs/blob/master/chef_master/source/data_collection_habitat.rst>`__
 
 
-The data collection capabilities of Chef Automate can also be used to collect and report on Habitat ring data. The Prism Habitat package collects this data and sends it to a Chef Automate server's REST API endpoint. You can configure settings like the data collector URL, token, the Habitat supervisor used to get the ring information, and so on. For more information on the Prism package, see :doc:`Habitat Prism </habitat_prism>`. For more information on Habitat, see the `Habitat site <https://habitat.sh/>`__.
+The data collection capabilities of Chef Automate can also be used to collect and report on Habitat ring data. The Prism Habitat package collects this data and sends it to a Chef Automate server's REST API endpoint. You can configure settings like the data collector URL, token, the Habitat supervisor used to get the ring information, and so on. For more information on the Prism package, see `Habitat Prism </habitat_prism.html>`__. For more information on Habitat, see the `Habitat site <https://habitat.sh/.html>`__.
```
Comment (davidwrede) [inline chef_master/source/data_collection_habitat.rst:6]: Because this topic is removed in a subsequent PR, make sure you merge this one in first.

Comment (brewn) [reply chef_master/source/data_collection_habitat.rst:6]: yep, holding off on the habitat one till this goes through.

File: chef_master/source/dsl_recipe_analytics.rst [hunk 7]
```diff
@@ -7,12 +7,12 @@ Recipe DSL for Chef Analytics
...
 
-          If you are a new Chef customer, or are looking to gain better insight into your fleet, try :doc:`Chef Automate </chef_automate>`. You'll get a graphical interface and query language that gives you insight into operational, compliance, and workflow events. Download Chef Automate `here <https://downloads.chef.io/automate/>`__.
+          If you are a new Chef customer, or are looking to gain better insight into your fleet, try `Chef Automate </chef_automate.html>`__. You'll get a graphical interface and query language that gives you insight into operational, compliance, and workflow events. Download Chef Automate `here <https://downloads.chef.io/automate>`__.
 
 
 .. end_tag
 
-The following methods are :doc:`part of the Recipe DSL <dsl_recipe>` and are specific to the Chef Analytics platform. Use them to define audit rules within a recipe. The audit rules are processed when the chef-client is run in audit-mode.
+The following methods are `part of the Recipe DSL <dsl_recipe.html>`__ and are specific to the Chef Analytics platform. Use them to define audit rules within a recipe. The audit rules are processed when the chef-client is run in audit-mode.
```
Comment (davidwrede) [inline chef_master/source/dsl_recipe_analytics.rst]: Missing `/` in link.

File: chef_master/source/upgrade_client.rst [hunk 8]
```diff
@@ -27,7 +27,7 @@ Considerations
 
 Preparing your chef-client system before upgrading to Chef server will enhance your Chef experience.  During a Chef server upgrade, all of the data is extracted, converted to a new format, and then uploaded. A large amount of data (cookbooks, nodes, etc.,) can increase the upgrade process significantly and extend your downtime.
 
-Managing your data prior to upgrading will improve experience upgrading and using Chef server and client. Following this list of client-based tasks prior to upgrading will expedite the upgrade process and mitigate many common issues.  For more information on upgrading to Chef server, see :doc:`Enterprise Chef to Chef 12 Upgrades. <upgrade_server_notes>`
+Managing your data prior to upgrading will improve experience upgrading and using Chef server and client. Following this list of client-based tasks prior to upgrading will expedite the upgrade process and mitigate many common issues.  For more information on upgrading to Chef server, see `Enterprise Chef to Chef 12 Upgrades. <upgrade_server_notes.html>`__
```
Comment (davidwrede) [inline chef_master/source/upgrade_client.rst]: Missing `/` in link.

**Pull request merged**